### ISSUE TITLE
Statistics Indexer cuts off the Realname/Gecos

### ISSUE THREAD

**Issue by MrBenC** [ircdocs/irc-stats#17]
Fortunately I recalled the name of the web site.  But I can't promise every network staff member can or will do the same.

`werewolf.uk.eu.chatlounge.net: *** Notice -- Client connecting: ircstats (~grabber@stats-grabber.ircdocs.horse) [149.56.47.33] {?} [Statistics retrieval bot for **http://stats.ircdocs.**]`

Proposal:

Change the realname/gecos to: "Stats retrieval bot for http://stats.ircdocs.horse"
Or even to: "Stat retrieval bot for http://stats.ircdocs.horse/"

Basically, ensure the entire URL is included.  I hope and expect this change and request to be trivial.

Ben

**Comment by DanielOaks** [comment]
Ah, thanks very much for letting me know! I'll definitely throw the URL to the start instead, and will take another look at where to throw my email addy as well (since I send that as part of the Realname too).

**Comment by DanielOaks** [comment]
Thanks, have fixed this in the grabber :)
